packages/example/test/api.ts

```diff
@@ -0,0 +1,66 @@
+import { SELF } from 'cloudflare:test'
+import { expect, it } from 'vitest'
+
+it('should return 404 on GET /', async () => {
+  const response = await SELF.fetch('https://domain/')
+  expect(response.status).toBe(404)
+})
+
+it ('should return 200 on GET /api/test', async () => {
+  const response = await SELF.fetch('https://domain/api/test')
+  expect(response.status).toBe(200)
+})
+
+// This should be 405, once comet supports that
```

Comment:
I think it should now be supported on main

Reply:
Nice! I've updated the test to cover it